### ISSUE TITLE
schema

### ISSUE THREAD

**Issue by biancamaginley** [robftg/Rocket-Launches#5]
Hey it looks like the schema doesn't have all the elements and attributes that we talked about Monday. Are you adding more to it? I'm trying to mark-up some of the source code.  @robftg

**Comment by biancamaginley** [comment]
I changed it up to match what was going on with my code, but I know you want to add more to it.

**Comment by robftg** [comment]
Thank you, all apologizes didn't have my laptop with me to pull and take a look at work, It looks good from my quick glance, will take a deeper dive

**Comment by biancamaginley** [comment]
Yo whats up with the way the schema looks right now

**Comment by frabbitry** [comment]
@biancamaginley @robftg (and @BMT45)
Do you guys want to meet from 1-2 tomorrow? My Writing Center shift goes from 12-1:30, but I'd be able to make the last half hour of the meeting. Or, if you guys are free, we could meet right after class from 3-4. 

I think that it might be a good idea to get on the same page about the schema and come up with a system for divvying up tasks so that it's easier to keep up with project checkpoints as the semester goes on. 

Also, project checkpoint 3 is due tomorrow, and I'd like to more officially check in with you guys.

**Comment by ebeshero** [comment]
@frabbitry @biancamaginley @robftg Actually the checkpoint is due on Friday! (Remember, what's in the column on the right is "due next time". But yes, you should be working on it... :-)

**Comment by frabbitry** [comment]
Yikes! Sorry about that, you're right @ebeshero 
I wrote "tomorrow" but I meant "Friday," because it makes more sense to meet with you guys before the checkpoint. It's been kind of a long day. 

I just want to make sure that you guys are on track for the checkpoint, that's all.

**Comment by biancamaginley** [comment]
1 sounds good for tomorrow.

**Comment by frabbitry** [comment]
Cool! @BMT45 will be in there and can help you guys with the schema until I get there. 

@robftg ?

**Comment by robftg** [comment]
Yeah I can make 1 tomorrow, all due apologies for the late reply I didn't have connection while at work
